app/src/main/res/layout-land/profile_reset_pin_activity.xml

```diff
@@ -78,35 +77,53 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent" />
 
-          <org.oppia.android.app.profile.ProfileInputView
-            android:id="@+id/input_pin"
+          <com.google.android.material.textfield.TextInputLayout
+            android:id="@+id/profile_reset_input_pin"
+            style="@style/TextInputLayout"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_marginStart="168dp"
             android:layout_marginTop="32dp"
             android:layout_marginEnd="168dp"
-            app:isPasswordInput="true"
+            android:hint="@{viewModel.isAdmin ? @string/profile_reset_pin_admin_pin_label : @string/profile_reset_pin_user_pin_label}"
```

Comment:
hint will go under `TextInputEditText`

Reply:
@anandwana001 There is issue here here i've tried to put it in TextInputEditText but when i run it
the `hint` does'nt get animated to top

Reply:
This is the problem with setting the label dynamically 
Ref - https://stackoverflow.com/questions/34198599/textinputlayout-has-no-effect-for-giving-hint-programmatically-in-edittext

Reply:
I tried shifting to `TextInputEditText ` and making one of the add a string directly in one. I got the below result. This seems something related with the data binding issue. 

<img src="https://user-images.githubusercontent.com/25057618/103398750-081f1680-4b64-11eb-8b4b-b48018a797d9.png" width=250 height=500 />

Reply:
@anandwana001 This is the same issue here too - https://github.com/oppia/oppia-android/pull/2349/files#r550414387

Reply:
can we fix this?

Reply:
Adding screenshots for better reference 
Using Hint in TextInputLayout | Using Hint in TextInputEditText
--- | ---
![image](https://user-images.githubusercontent.com/54740946/103399295-6220db80-4b66-11eb-8cd4-908e136b0fae.png) | ![image](https://user-images.githubusercontent.com/54740946/103399297-6b11ad00-4b66-11eb-9580-331bb49e4caa.png)

Reply:
> can we fix this?

@anandwana001 fix to use hint in `TextInputEditText`?

Reply:
I think using hint inside TextInputLayout for all files would be better.

Reply:
could you confirm this from the Material design doc or Android official doc for better practice.

Reply:
@anandwana001 adding references that were guided by Material Design And Android Official Docs
Material design - https://material.io/components/text-fields/android#using-text-fields
Material Design Getting started Docs - https://github.com/material-components/material-components-android/blob/master/docs/getting-started.md#5-add-a-material-component-to-your-app

![image](https://user-images.githubusercontent.com/54740946/103404413-70c4be00-4b79-11eb-90fa-963561488ae3.png)

Reply:
okay, we can do it correctly in this PR and for other, I see there is only 1 merged PR so we can correct it in this PR only. 
Thanks @FareesHussain for correctness.

Reply:
shall I create a good first issue for this for your merged PR

Reply:
it's a line shifting, we can do in this PR only.

Reply:
Okay